docs/modules/install/pages/checklist.adoc

```diff
@@ -13,7 +13,7 @@ As a technopolitical project, Decidim needs several things to work. This is a no
 .. Configure *redirection from HTTP to HTTPS* on your web server.
 .. Configure your *Certificate Authority Authorization* (CAA) DNS records
 .. Install complete *Certificate Chains* if it is needed for your provider
-.. Use current SSL/TLS Protocols (*TLS 1.2 or 1.3*)
+.. Use [current SSL/TLS Protocols](https://stellastra.com/cipher-suite) (*TLS 1.2 or 1.3*)
```

Comment:
This is just a list that doesn't make much sense. Just checking the other elements of the list and I see that we already have this a couple of lines after:

>    .. Use the [https://www.ssllabs.com/ssltest/[SSL](https://www.ssllabs.com/ssltest/%5BSSL) Server Test] and follow their recommendations

So, maybe this particular recommendation of the versions isn't necessary? I mean, this recommendation will change and we can just "outsource" the recommendation of protocols to SSLLabs. What do you think?

Reply:
Thanks, I have just modified the page itself to include a description clarifying its contents. It is now clear what the page is for, being a database of TLS cipher suites, with each cipher linking to its own breakdown by key-authentication and authentication mechanism, cipher, and hash. For example, why TLS_DH_RSA_WITH_3DES_EDE_CBC_SHA is discouraged.

Reply:
Sorry, we'll not go forward with this proposed change. As I mentioned, the idea here is to have a really fast way of checking out what is necessary to have configured in Decidim, and we don't want to have too much information here.

I'll drop this line in another PR as it should be detected by the scanner in SSLLabs as I mentioned in my other comment.

Mind that the target implementer of Decidim is not an expert in SSL, nor an expert System Administrator/DevOps, so we need to keep it easy to them. If I'm a newcomer to Decidim, and go to your link and click some of the explanations, I'd said "this is too difficult for me, I can't work with Decidim". 

I appreciate the effort on having this list for sysadmins, but I don't think Decidim is the place to link it to.